.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "Traffic dev env",
+	"image": "ghcr.io/xoolive/traffic/dev:latest",
+	// "build": {
+	// 	"dockerfile": "Dockerfile",
+	// 	"context": ".."
+	// },
+
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
+				"ms-python.python",
+				"ms-toolsai.jupyter"
+			]
+		}
+	},
+	// do a pip install of the traffic library to avoid confusing importlib-metadata:
+	// "importlib_metadata.PackageNotFoundError: No package metadata was found for traffic"
+	"postCreateCommand": "pip install -e ."
```

Comment:
I don't get why it is not `poetry install`?
Or is it because the proper python environment is not loaded (yet) with the current configuration? Shouldn't we rather look at pointing to the freshly created poetry environment?

Reply:
The `poetry install` happens in the container and all the dependencies (except for traffic itself) are already available in the container. The `pip install -e .` just installs the local traffic package (the one you're working on) visible.
If you have an idea how to get around that, feel free to add it. But I don't know how else to deal with the package metadata not found error...